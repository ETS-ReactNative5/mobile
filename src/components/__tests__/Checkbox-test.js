import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Checkbox from '../Checkbox'

it('renders', () => {
  const tree = renderer.create(
    <Checkbox />
  ).toJSON();
  expect(tree).toMatchSnapshot()
})

it('renders background color when checked', () => {
  const tree = renderer.create(
    <Checkbox selected={true}/>
  ).toJSON();
  expect(tree).toMatchSnapshot()
})
