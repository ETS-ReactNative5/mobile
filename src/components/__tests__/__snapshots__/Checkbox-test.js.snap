// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<CheckboxField
  checkboxStyle={undefined}
  containerStyle={undefined}
  defaultColor="white"
  onSelect={undefined}
  selected={undefined}
  selectedColor="#1995ea"
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        undefined,
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
</CheckboxField>
`;

exports[`renders background color when checked 1`] = `
<CheckboxField
  checkboxStyle={undefined}
  containerStyle={undefined}
  defaultColor="white"
  onSelect={undefined}
  selected={true}
  selectedColor="#1995ea"
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        undefined,
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
</CheckboxField>
`;
